dbcrossbar/src/clouds/gcloud/client.rs

```diff
@@ -222,6 +222,68 @@ impl Client {
         }
     }
 
+    /// Make an HTTP POST request and return the response.
+    async fn post_helper<Body>(
+        &self,
+        url: &Url,
+        body: &Body,
+    ) -> Result<reqwest::Response, ClientError>
+    where
+        Body: fmt::Debug + Serialize + std::marker::Sync,
+    {
+        trace!("POST {}", url);
+        let token = self.token().await?;
+        let wait_options = WaitOptions::default()
+            .backoff_type(BackoffType::Exponential)
+            .retry_interval(Duration::from_secs(10))
+            // Don't retry too much because we're probably classifying some
+            // permanent errors as temporary.
+            .allowed_errors(3);
```

Comment:
```suggestion
            .retry_interval(Duration::from_secs(1))
            // Don't retry too much because we're probably classifying some
            // permanent errors as temporary.
            .allowed_errors(5);
```

Allow an earlier retry, because Google Cloud errors tend to be extremely transient.

Reply:
updated